### ISSUE TITLE
Combine view of attributes "on token mouse-over" with "directly in text"

### ISSUE THREAD

**Issue by Ansa211** [czcorpus/kontext#2168]
Current implementation:
Go to View > Corpus-specific settings
In the Viewing options, there is a choice between three modes:
- directly in text - for KWIC tokens only
- directly in text - for each token
- on token mouse-over
Only one of the three modes is active at any given moment; 
the choice of attributes to be shown is independent of the selected mode.

Suggested behaviour:
let all three modes apply simultaneously and make the choice of attributes to be shown separate for each mode.

Example situation:
I would like to see the word and pos of each token in the text;
additionally, I want to see the p_lemma of the KWIC tokens;
finally, I want to see the tag, p_lemma and ep_lemma on token mouse-over of any token.
In other words, more attributes are shown in text for the KWIC than for words in the context,
and even more attributes may be accessed by the mouse-over.

**Comment by tomachalek** [comment]
I have to discuss this with @michkren (and possibly with other colleagues) as this deviates a lot from the original (No)SkE implementation where only a single set of attributes is used in both URL arguments and user interface.

**Comment by Ansa211** [comment]
I think we can close this one; the currently available option to combine directly in text on KWIC and on token mouseover for other attributes seems to strike the balance between simplicity and functionality quite nicely!
